### ISSUE TITLE
[BUG] import phoenix as px , fails with PydanticSchemaGenerationError

### ISSUE THREAD

**Issue by DixitAdh** [Arize-ai/phoenix#4243]
**Describe the bug**
Latest version of arize-phoenix package import is failing with below error

```bash
INFO:py4j.clientserver:Received command c on object id p0
PydanticSchemaGenerationError: Unable to generate pydantic-core schema for typing_extensions.NoDefault. Set `arbitrary_types_allowed=True` in the model_config to ignore this error or implement `__get_pydantic_core_schema__` on your type to fully support it.

If you got this error by calling handler(<some type>) within `__get_pydantic_core_schema__` then you likely need to call `handler.generate_schema(<some type>)` since we do not call `__get_pydantic_core_schema__` on `<some type>` otherwise to avoid infinite recursion.

For further information visit https://errors.pydantic.dev/2.5/u/schema-for-unknown-type
File <command-3555972235464747>, line 1
----> 1 import phoenix as px
      2 from phoenix.trace.langchain import LangChainInstrumentor
      4 session = px.launch_app()
File /local_disk0/.ephemeral_nfs/envs/pythonEnv-4cb5b38c-3235-48e5-b81d-07e4a7eb36aa/lib/python3.10/site-packages/pydantic/_internal/_generate_schema.py:366, in GenerateSchema._unknown_type_schema(self, obj)
    365 def _unknown_type_schema(self, obj: Any) -> CoreSchema:
--> 366     raise PydanticSchemaGenerationError(
    367         f'Unable to generate pydantic-core schema for {obj!r}. '
    368         'Set `arbitrary_types_allowed=True` in the model_config to ignore this error'
    369         ' or implement `__get_pydantic_core_schema__` on your type to fully support it.'
    370         '\n\nIf you got this error by calling handler(<some type>) within'
    371         ' `__get_pydantic_core_schema__` then you likely need to call'
    372         ' `handler.generate_schema(<some type>)` since we do not call'
    373         ' `__get_pydantic_core_schema__` on `<some type>` otherwise to avoid infinite recursion.'
    374     )
```

**To Reproduce**
1. pip install arize-phoenix 
2. import phoenix as px
3. See error

**Expected behavior**
Package should be imported without an error

**Screenshots**
<img width="1244" alt="image" src="https://github.com/user-attachments/assets/e13bb296-0448-4c13-b022-63bd77be9e6e">

**Environment (please complete the following information):**
 - Notebook Databricks
 - Version [e.g. 0.0.1]

**Comment by undo76** [comment]
Till this issue is solved, as a workaround you can pin `typing-extensions = "<4.12"`

**Comment by RogerHYang** [comment]
> typing-extensions = "<4.12"

I just tried it in databricks and it does work. Thank you, @undo76!

**Comment by axiomofjoy** [comment]
This issue is resolved on `arize-phoenix==4.27.0`. Please upgrade or ensure your `pydantic` version is not in the range `2.4.1<=pydantic<=2.7.1`.

See https://github.com/Arize-ai/phoenix/issues/4310 for details.
